cmd/thriftlint/lint.go

```diff
@@ -0,0 +1,193 @@
+package main
```

Comment:
Moving much of the logic into a library is the intention. It'll make it easier to build Phab/whatever integration.